source/Calamari.Shared/Integration/Scripting/WindowsPowerShell/Bootstrap.ps1

```diff
@@ -569,7 +569,12 @@ function Initialize-ProxySettings() {
 		if ($useDefaultProxy) {
 			# The system proxy should be provided through an environment variable, which has been used to initialize $proxyHost
 			if ($proxyUri -ne $null) {
-				$proxy = New-Object System.Net.WebProxy($proxyUri)
+				if (-not (Test-Path variable:IsWindows) -or $isWindows){
```

Comment:
It's not obvious that `$isWindows` is only defined for NetCore, please add a comment to explain.